java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -869,28 +869,75 @@ public static HostColumnVector timestampNanoSecondsFromBoxedLongs(Long... values
     private boolean nullable;
     private long rows;
     private long estimatedRows;
+    private long rowCapacity = 0L;
+    private long validCapacity = 0L;
     private boolean built = false;
     private List<ColumnBuilder> childBuilders = new ArrayList<>();
+    private Runnable nullHandler;
 
     private int currentIndex = 0;
-    private int currentByteIndex = 0;
-
+    private long currentByteIndex = 0;
```

Comment:
Why did `currentByteIndex` change to a long?  This is exposed publicly through a method and that causes this to be a breaking API change. None of our code calls into that method right now, so I would propose that we deprecate it.  We also only use this for strings, we set it in lots of places, but only read it for strings. CUDF has a limit of 2GiB of string data per column so an int should be able to handle it.  I would propose that we rename this to currentStringByteIndex, and only ever tough it in the case of strings (because that code is not common any more)

Reply:
Yes. I reverted the change and removed unnecessary usages of `currentByteIndex`. For now, `currentByteIndex` is a string specialized pointer.